samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -840,17 +840,17 @@ For example, for a device with the device ID ``698d4c11-0ccc-4f04-89cd-6882724e3
 
 .. tabs::
 
-   .. group-tab:: Bash
+   .. group-tab:: MQTT
 
       .. code-block:: console
 
-        west build --board nrf5340dk/nrf5340/cpuapp/ns -p always -- -DSHIELD=nrf7002ek -DEXTRA_CONF_FILE=overlay_nrf7002ek_wifi_no_lte.conf -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y -DCONFIG_NRF_CLOUD_CLIENT_ID="698d4c11-0ccc-4f04-89cd-6882724e3f6f"
+         west build --board nrf5340dk/nrf5340/cpuapp/ns -p always -- -DSHIELD=nrf7002ek -DEXTRA_CONF_FILE=overlay_nrf7002ek_wifi_no_lte.conf -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y -DCONFIG_NRF_CLOUD_CLIENT_ID="698d4c11-0ccc-4f04-89cd-6882724e3f6f"
 
-   .. group-tab:: PowerShell
+   .. group-tab:: CoAP
 
       .. code-block:: console
 
-         west build --board nrf5340dk/nrf5340/cpuapp/ns -p always -- -DSHIELD=nrf7002ek "-DEXTRA_CONF_FILE=overlay_nrf7002ek_wifi_no_lte.conf" -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y  "-DCONFIG_NRF_CLOUD_CLIENT_ID=\`"698d4c11-0ccc-4f04-89cd-6882724e3f6f\`""
+         west build --board nrf5340dk/nrf5340/cpuapp/ns -p always -- -DSHIELD=nrf7002ek -DEXTRA_CONF_FILE=overlay_nrf7002ek_wifi_coap_no_lte.conf -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y -DCONFIG_NRF_CLOUD_CLIENT_ID="698d4c11-0ccc-4f04-89cd-6882724e3f6f"
```

Comment:
It would probably be good to document somewhere how to properly provide string consts to KConfig in powershell. It isn't at all obvious, and made more complicated by how cursed the KConfig parser is, since the quotes themselves have to be escaped quotes inside quotes

Reply:
In fact, regarding that, I think quite possibly we need to change `-DCONFIG_NRF_CLOUD_CLIENT_ID="698d4c11-0ccc-4f04-89cd-6882724e3f6f"` to `"-DCONFIG_NRF_CLOUD_CLIENT_ID=\"698d4c11-0ccc-4f04-89cd-6882724e3f6f\""` even on linux? Or so has been my personal experience since switching

Reply:
We provide a PowerShell example in the next section. But I'll go ahead and do it here too, and add quotes for Bash, because I had to use them myself.

Reply:
> But I'll go ahead and do it here too,

Up to you, it's honestly not a horrible idea to simplify this example to just CoAP vs MQTT -- I only point this out more as a problem we have more broadly with NCS and string-type KConfig args